Shokofin/Resolvers/Models/LinkGenerationResult.cs

```diff
@@ -111,6 +126,10 @@ public void Print(ILogger logger, string path) {
             FixedSubtitles = a.FixedSubtitles + b.FixedSubtitles,
             SkippedSubtitles = a.SkippedSubtitles + b.SkippedSubtitles,
             RemovedSubtitles = a.RemovedSubtitles + b.RemovedSubtitles,
+            CreatedAudioFiles = a.CreatedAudioFiles + b.CreatedAudioFiles,
+            FixedAudioFiles = a.FixedAudioFiles + b.FixedAudioFiles,
+            SkippedAudioFiles = a.SkippedAudioFiles + b.SkippedAudioFiles,
+            RemovedAudioFiles = a.RemovedAudioFiles + b.RemovedAudioFiles,
```

Comment:
I'd probably merge Subtitles and Audio handling into one flow, since they're sharing the same matching rules except extensions. Would let avoid a repetition of 50+ LoC. WDYT @revam?

Reply:
You can either leave it as-is, or refactor the subtitle/audio handling to be 'external file' handling. I'll leave it to you what you want to do, as I'm fine with either solution.